.github/workflows/ghcr.yml

```diff
@@ -304,10 +371,10 @@ jobs:
   # Push the runtime Docker images to the ghcr.io repository
   ghcr_push_runtime:
     runs-on: ubuntu-latest
-    needs: [ghcr_build_runtime, test_runtime, runtime_integration_tests_on_linux]
+    needs: [ghcr_build_runtime_amd64, test_runtime, runtime_integration_tests_on_linux]
```

Comment:
This job runs for both platforms amd64 and arm64 so I don't think you can only wait for the amd64 one